### ISSUE TITLE
Warning messages in LOG : "has no existing object"

### ISSUE THREAD

**Issue by Berchemer** [Jey-Cee/ioBroker.net-tools#24]
I think it is a device I added manually. Seems to have all data as the other devices. Don't know how I could solve the problem.

```
net-tools.0 | 2023-02-07 07:55:30.382 | warn | State "net-tools.0.xyz.rps" has no existing object, this might lead to an error in future versions
net-tools.0 | 2023-02-07 07:55:30.382 | warn | State "net-tools.0.xyz.time" has no existing object, this might lead to an error in future versions
net-tools.0 | 2023-02-07 07:55:30.381 | warn | State "net-tools.0.xyz.alive" has no existing object, this might lead to an error in future versions

```

**Comment by Jey-Cee** [comment]
This message happens if a device was deleted without a adapter restart.

So you are sure this is a device you have newly added?

**Comment by Jey-Cee** [comment]
Fixed with v1.0.4
